packages/spawner/scripts/build-bundle-spawn.js

```diff
@@ -0,0 +1,33 @@
+/* global __dirname */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'ses';
+import fs from 'fs';
+import process from 'process';
+import bundleSource from '@agoric/bundle-source';
+
+async function writeSourceBundle(contractFilename, outputPath) {
+  await bundleSource(contractFilename).then(bundle => {
+    // TODO: fix
+    // @ts-ignore mkdirSync believes it only accepts 2 arguments.
+    fs.mkdirSync(`${__dirname}/../bundles`, { recursive: true }, err => {
```

Comment:
`fs.mkdirSync` *does* only accept 2 arguments, and it throws on error.  The callback is only used for `fs.mkdir`.

Reply:
ah, I copied that from `zoe/scripts/build-zcfBundle.js`.. I'll make a PR to fix it over there too.